components/transfer/TransferConfirmModal.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <NeoModal v-model="isModalActive" scroll="clip" @close="emit('close')">
+    <div
+      class="modal-width is-flex is-flex-direction-column is-justify-content-space-between">
+      <div>
+        <div
+          class="is-flex is-justify-content-center is-align-items-center header px-6 py-3">
+          <span class="is-size-5 has-text-weight-bold">
+            {{ $t('teleport.send') }}
+            <span class="is-uppercase">{{ unit }}</span>
+          </span>
+          <NeoIcon
+            class="position-right is-clickable mr-6"
+            icon="close"
+            pack="fas"
+            @click.native="closeModal" />
+        </div>
+        <div
+          class="px-6 is-scrollable"
+          :class="{
+            'is-bordered-top': isExpandList,
+          }">
+          <div
+            class="is-flex is-justify-content-space-between is-align-items-center py-4">
+            <span class="has-text-weight-bold is-size-6">{{
+              $t('transfers.network')
+            }}</span>
+            <div class="is-flex is-align-items-center">
+              <img class="mr-2 is-24x24" :src="tokenIcon" alt="token" />
```

Comment:
`is-24x24` does not work like that you need to add `image` class ref [Bulma doc](https://bulma.io/documentation/elements/image/#fixed-square-images)

Just saw same has been used in Transfer component `is-20x20` doesn't exist

Reply:
Thanks. I did miss this one.